Observer/Order/Save.php

```diff
@@ -134,7 +138,8 @@ public function execute(Observer $observer)
                 if ($nostoCustomer instanceof NostoCustomer === false) {
                     return;
                 }
-                $orderService = new OrderConfirm($nostoAccount);
+                $store = $order->getStore();
+                $orderService = new OrderConfirm($nostoAccount, $this->nostoHelperUrl->getActiveDomain($store));
```

Comment:
This throws an exception, please add to the docblock as `* @throws \Nosto\NostoException` or try-catch it.